holoviews/ipython/jsslider.jinja

```diff
@@ -32,31 +62,29 @@
             else if(this.slider_ids.length == 1) { key += ',';}
         }
         key += ")";
-        this.current_frame = this.keyMap[key];
-        if(this.current_frame == undefined) {
-            $("#" + this.img_id).html(this.notFound);
-        } else if(this.load_json) {
-            var data_url = "{{ server }}/" + this.fig_id + "/" + this.current_frame;
-		    if(this.mpld3) {
-                d3.select("#"+this.img_id).selectAll("*").remove();
-				$.getJSON(data_url,
-                    (function(img_id) {
-					     return function(data) {
-                             mpld3.draw_figure(img_id, data);
-					     };
-				     }(this.img_id))
-				);
-			} else {
-                $("#" + this.img_id).load("{{ server }}/" + this.fig_id + "/" + this.current_frame);
-            }
-		}else {
-		    if(this.mpld3) {
-                d3.select("#" + this.img_id).selectAll("*").remove();
-                mpld3.draw_figure(this.img_id, this.frames[this.current_frame]);
-            }else {
-                $("#" + this.img_id).html(this.frames[this.current_frame]);
-            }
-        }
+	if(this.cached) {
+	    this.current_frame = this.keyMap[key];
+            this.update()
+	} else {
+	    var kernel = IPython.notebook.kernel;
```

Comment:
You didn't see to need ``IPython.notebook.kernel`` before (you used methods on something called ``mpld3``). Does this mean the mechanism by which this works has fundamentally changed?